packages/codemirror-extension/src/index.ts

```diff
@@ -236,20 +236,23 @@ function activateEditorCommands(
       (settings.get('lineWiseCopyCut').composite as boolean) ?? lineWiseCopyCut;
   }
 
+  const editorOptions: any = {
+    keyMap,
+    theme,
+    scrollPastEnd,
+    styleActiveLine,
+    styleSelectedText,
+    selectionPointer,
+    lineWiseCopyCut
+  };
+
   /**
    * Update the settings of the current tracker instances.
    */
   function updateTracker(): void {
     tracker.forEach(widget => {
       if (widget.content.editor instanceof CodeMirrorEditor) {
-        const { editor } = widget.content;
-        editor.setOption('keyMap', keyMap);
-        editor.setOption('lineWiseCopyCut', lineWiseCopyCut);
-        editor.setOption('scrollPastEnd', scrollPastEnd);
-        editor.setOption('selectionPointer', selectionPointer);
-        editor.setOption('styleActiveLine', styleActiveLine);
-        editor.setOption('styleSelectedText', styleSelectedText);
-        editor.setOption('theme', theme);
+        widget.content.editor.setOptions(editorOptions);
```

Comment:
It looks like `editorOptions` is never updated, so this `setOptions` command is never really updating options.

Reply:
Fixed in #10353